pkg/ebpf/common.go

```diff
@@ -181,3 +184,9 @@ func isRHELOrCentOS() (bool, error) {
 	}
 	return isCentOS(platform) || isRHEL(platform), nil
 }
+
+// hasOldKernelAPI compares current kernel version to the minimum kernel version that uses the new API
+func hasOldKernelAPI(currentKernelCode uint32) bool {
```

Comment:
`requiresPre410Kernel`

Reply:
Actually, what about `hasKernelOlderThan4_1_0`

Reply:
Yea just about to point out using `has` looks good. I'd rather keep `410` instead of underscores it's more golang that way?